assets/xml/objects/object_market_obj.xml

```diff
@@ -1,65 +1,69 @@
+        <Texture Name="gWestClockTownMarketGoldPatternedWall5Tex" OutName="west_clock_town_market_gold_patterned_wall_5" Format="ci4" Width="64" Height="64" Offset="0xDDE0" />
+        <Texture Name="gWestClockTownMarketSilverPatternedWall3Tex" OutName="west_clock_town_market_silver_patterned_wall_3" Format="ci4" Width="64" Height="64" Offset="0xE5E0" />
+        <Texture Name="gWestClockTownMarketSilverPatternedWall4Tex" OutName="west_clock_town_market_silver_patterned_wall_4" Format="ci4" Width="64" Height="64" Offset="0xEDE0" />
+        <Texture Name="gWestClockTownMarketSilverPatternedWall5Tex" OutName="west_clock_town_market_silver_patterned_wall_5" Format="ci4" Width="64" Height="64" Offset="0xF5E0" />
+        <Texture Name="gWestClockTownMarketSilverPatternedWall6Tex" OutName="west_clock_town_market_silver_patterned_wall_6" Format="ci4" Width="64" Height="64" Offset="0xFDE0" />
+        <Texture Name="gWestClockTownMarketPoleTex" OutName="west_clock_town_market_pole" Format="rgba16" Width="32" Height="16" Offset="0x105E0" />
+        <Texture Name="gWestClockTownMarketFloorWithEdgeTex" OutName="west_clock_town_market_floor_with_edge" Format="rgba16" Width="32" Height="64" Offset="0x109E0" />
+        <Texture Name="gWestClockTownMarketStairRiserTex" OutName="west_clock_town_market_stair_riser" Format="rgba16" Width="32" Height="16" Offset="0x119E0" />
+        <Texture Name="gWestClockTownMarketFlowerBedTopTex" OutName="west_clock_town_market_flower_bed_top" Format="rgba16" Width="32" Height="32" Offset="0x11DE0" />
+        <Texture Name="gWestClockTownMarketFlowerBedSidesTex" OutName="west_clock_town_market_flower_bed_sides" Format="rgba16" Width="32" Height="32" Offset="0x125E0" />
+        <Texture Name="gWestClockTownMarketSafeTex" OutName="west_clock_town_market_safe" Format="rgba16" Width="32" Height="32" Offset="0x12DE0" />
+        <Texture Name="gWestClockTownMarketRugTex" OutName="west_clock_town_market_rug" Format="rgba16" Width="64" Height="16" Offset="0x135E0" />
+        <Texture Name="gWestClockTownMarketBankCurtainTex" OutName="west_clock_town_market_bank_curtain" Format="ia8" Width="32" Height="32" Offset="0x13DE0" />
+        <Texture Name="gWestClockTownMarketCowFrontTex" OutName="west_clock_town_market_cow_front" Format="ci8" Width="8" Height="8" Offset="0x141E0" />
+        <Texture Name="gWestClockTownMarketCowBodyTex" OutName="west_clock_town_market_cow_body" Format="i8" Width="8" Height="8" Offset="0x14220" />
+        <Texture Name="gWestClockTownMarketBankAdvertisementsTex" OutName="west_clock_town_market_bank_advertisements" Format="i4" Width="64" Height="32" Offset="0x14260" />
+        <Texture Name="gWestClockTownMarketSafeWheelAndHandleTex" OutName="west_clock_town_market_safe_wheel_and_handle" Format="rgba16" Width="16" Height="16" Offset="0x14660" />
+
+        <!-- West Clock Town Market DisplayLists -->
+        <DList Name="gWestClockTownMarketBankAdvertisementsAndFakeDoorNightDL" Offset="0x18C60" />
```

Comment:
More so just out of curiosity, what do you mean by fake door?

Reply:
I have the same question

Reply:
Behind the bank vaults, there's this fake door:
![image](https://user-images.githubusercontent.com/12245827/191570284-86d63feb-e63c-48db-a3db-3d18f4652306.png)

I call it "fake" because it's not an actual door; it's just a picture of one. You can't interact with it in any way, and in fact you can't even see it without going out of bounds. I didn't want people mistaking this for a texture for a real, working door, hence the name. However, I'm happy to change it, since I kind of don't care very much about the name of a texture you can't even see under normal circumstances.

Reply:
Oh, it's off to the side in the banker's kiosk?

Reply:
Correct. You can't see it because the vaults are in front of it. But it's just off to the side behind the banker.

Reply:
I think I specifically don't want `fake door` because that sounds like DoorKiller in OoT. Possibly just door would be fine, with a note that it's hidden in-game.